advance_check_void/README.rst

```diff
@@ -0,0 +1,76 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+========
+Overview
+========
+
+This module is used to make vendor bill payments via checks and also void the checks. 
+With this module we can keep a track of payment check History and also see all voided Checks.
```

Comment:
The README should be automatically generated.
Please add the fragments in a README subdirectory.

Reply:
...maybe not in the OSI repo.

Reply:
I'll add it anyway just to have it in there, that way if the repo is ever updated to generate them or it goes to OCA it will already be done.